src/mappers.ts

```diff
@@ -218,7 +209,7 @@ export async function getInputParams() {
     }
     if(config.properties != undefined)
     {
-        var propFile = config.properties.userPropertyFile;
+        var propFile = config.properties[0].userPropertyFile;
```

Comment:
User property is not an array. There was a mistake in the syntax shared by me. This is the correct syntax - 
properties:
  userPropertyFile: peak-load.properties
  
Can we revert this change?

Reply:
understood